### DISCUSSION TITLE
Snappy package doesn't support Python 3.11

### DISCUSSION THREAD

**Discussion by ahmetanbar** [aio-libs/aiokafka#931] [General]
Hello,

I am new at Kafka. I want to consume messages which are compresses by snappy. I see that aiokafka automatically decompress messages. So that, snappy package should be installed. 

When I try to install `python-snappy` package as how it is said [here](https://aiokafka.readthedocs.io/en/stable/index.html#optional-snappy-install)
`pip install 'aiokafka[snappy]'`
or
via `pip install python-snappy`, I got error 
```
Building wheel for python-snappy (setup.py) ... error
  ERROR: Command errored out with exit status 1:
...
src/snappy/snappymodule.cc:33:10: fatal error: 'snappy-c.h' file not found
```

It is due to this package doesn't have wheels for Python 3.11. An [issue](https://github.com/andrix/python-snappy/issues/124) exists at the project regarding this.

I think I can apply those solutions to solve the issue:
1. Build and install package manually: I don't want to do this
2. Wait wheels publish for 3.11 from the package: I don't know when it will be available
3. Get compressed raw message without any decompression by aiokafka and decompress it manually by using other packages like [that](https://pypi.org/project/cramjam/): It is what I prefer.

Is the third one possible right now?

I am looking for your helps and ideas.

Thank you

**Comment by ods** [comment]
You need to install snappy headers before installing `python-snappy` from source package. On Mac OS it can be done with `brew install snappy` (see [docs](https://github.com/andrix/python-snappy#frequently-asked-questions)).

BTW, there is plan to move to cramjam in aiokafka. Primarily because `python-snappy` is not supported anymore. But it's not the first in the agenda, sorry.

**Comment by ahmetanbar** [comment]
Thanks for your answer

**Reply by ods** [reply]
Hi @ahmetanbar,
Could you please check with this branch https://github.com/aio-libs/aiokafka/pull/940, replacing python-snappy and zstandard dependencies with cramjam? I'm especially interested to verify it with real-live applications using aiokafka with Snappy and/or ZStandard compression.
